README.md

```diff
@@ -176,6 +176,24 @@ This is controlled by the property: `primary-meta-store.access-control-type` It
 | `READ_AND_WRITE_AND_CREATE_ON_DATABASE_WHITELIST`     | Reads are allowed, writes are allowed on database names listed in the `primary-meta-store.writable-database-white-list` property, creating new databases is allowed and they are added to the white-list automatically. |
 | `READ_AND_WRITE_ON_DATABASE_WHITELIST`                | Reads are allowed, writes are allowed on database names listed in the `primary-meta-store.writable-database-white-list` property, creating new databases is not allowed. |
```

Comment:
this needs to be updated as the writable-database-white-list can now also be set on the federated metastore correct?